services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -838,6 +838,16 @@ public List<Boolean> snappingIncludeClosuresList() {
   @Nullable
   public abstract Boolean enableRefresh();
 
+  /**
+   * Whether to return calculated toll cost for the route.
+   * If null, default is false.
+   *
+   * @return true if the request includes toll cost
+   */
```

Comment:
replace javadoc with comments to make API undocumented cc @zugaldia 
```suggestion
  /*
   * Whether to return calculated toll cost for the route.
   * If null, default is false.
   *
   * return true if the request includes toll cost
   */
```